cluster/gce/trusty/kube-manifest/fluentd-es.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
```

Comment:
An alternative: for the manifest files without salt configuration, such as these three files, we can simply copy them from cluster/saltbase/salt when creating the kube-manifest.tar.gz. So, the dir cluster/gce/trusty/kube-manifest only contains the manifests containing salt configurations, such as  etcd, api-server. This should be used by both CoreOS and trusty, so I am fine to move it to be cluster/gce/kube-manifest. Does it solve your concern?